docs/build/js-client/10-marine-js.md

```diff
@@ -2,11 +2,36 @@
 
 > This section is experimental as the API is not stable and probably will change.
 
+## What is Marine JS
+
+### Intro
+When JS client starts up, it starts with initialization Marine JS runtime.
+The runtime hosts wasm services and even allows you to register your own wasm services.
+For example, AquaVM service for processing particles and Air code resides in Marine JS.
+That's why Marine JS is a foundation of JS client. 
```

Comment:
this is all fine but why does that matter to the developer we're trying to onbard? more imprtantly, how does this relate to the Rust peer and why/when does it matter?

Reply:
Why it should relate to Rust peer somehow? JS peer completely different.
Well, if developer clicked on Marine JS section in the doc, then he would like to know about it, i hope. This 4 sentences give a basic knowledge about Marine JS to understand decisions we made in JS client and how things work internally.

Reply:
> The runtime hosts wasm services and even allows you to register your own wasm services
i think that's what caused me to pause as that's not exactly true and any analogies the developer will draw from prior learning experiences, which are all based on distributed peers and services. maybe the section shouldn't be called Marine JS but Extending JS Client With Marine Modules